update faculty in HCI

xliucs opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

xliucs committed:
update faculty in HCI

emeryberger commented:
There are a number of spurious #NAME? entries (see your diff). Please fix.

Also, Anind Dey is not listed as a CSE adjunct faculty member, so AFAICT he does not meet the inclusion criteria. Please provide supporting evidence otherwise.

xliucs committed:
Fixed issues in HCI & #NAME?

xliucs commented:
Hi  Emery,

I just realize that Anind is not an adjunct faculty in CSE, so I just removed him. I also fixed the #NAMES? issues. Please let me know if there is any problem. Thanks!

emeryberger commented:
The `dblp-aliases.csv` file is also large big a diff. Please revert and only change the necessary lines (it is almost certainly a line ending issue).

xliucs committed:
fixed aliases
